src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form

```diff
@@ -80,14 +75,57 @@
       </component>
       <component id="8234b" class="javax.swing.JLabel" binding="connectionResults">
         <constraints>
-          <grid row="3" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+          <grid row="3" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties>
           <enabled value="true"/>
           <font name="Courier 10 Pitch"/>
           <text value=""/>
         </properties>
       </component>
+      <component id="1c8d6" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="4" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="Npm exclusions"/>
+          <toolTipText value="Glob pattern of excluded paths from Xray scan for npm."/>
```

Comment:
Glob pattern of excluded paths from Xray scan for npm. -->
Pattern of project paths to exclude from Xray scanning for npm.